runtime/Cargo.toml

```diff
@@ -114,7 +114,6 @@ frozen-abi = [
     "solana-accounts-db/frozen-abi",
     "solana-compute-budget/frozen-abi",
     "solana-cost-model/frozen-abi",
-    "solana-perf/frozen-abi",
```

Comment:
Do you want to comment this instead of deleting it like the other removals?

Reply:
Sure, thanks for mentioning it. I forgot